src/index.html

```diff
@@ -5,9 +5,54 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__image"
+        src="./images/imac.jpeg" 
+        alt="Mac"
+        >
+      <h1 class="card__title">
+          APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__product-code">
+          Product code: 195434
+      </p>
+      <div class="card__ratting">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__reviews">
+            Reviews: 5
+        </p>
+      </div>
+      <div class="card__price">
```

Comment:
Add a blank line between the blocks
```suggestion
      </div>
      
      <div class="card__price">
```